docs/sources/reference/components/loki.source.awsfirehose.md

```diff
@@ -65,6 +65,20 @@ The component will start an HTTP server on the configured port and address with
 
 - `/awsfirehose/api/v1/push` - accepting `POST` requests compatible with [AWS Firehose HTTP Specifications](https://docs.aws.amazon.com/firehose/latest/dev/httpdeliveryrequestresponse.html).
 
+You can use the [X-Amz-Firehose-Common-Attributes](https://docs.aws.amazon.com/firehose/latest/dev/httpdeliveryrequestresponse.html) header to set extra static labels.
+Label names must be prefixed with `lbl_`. 
+Label names and label values must be compatible with the [Prometheus data model](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels) specification.
+
+Example of the valid `X-Amz-Firehose-Common-Attributes` value with two custom labels:
```

Comment:
When I read this, it wasn't immediately apparent whether these labels are set on the component side or on the AWS firehose side and the component is just reading them out.

Can we make it more explicit and explain where these labels will be applied to (iiuc the read log lines)?